seed/static/seed/js/controllers/data_quality_admin_controller.js

```diff
@@ -222,8 +222,28 @@ angular.module('BE.seed.controller.data_quality_admin', [])
       var get_configured_rules = function () {
         var rules = [];
         var misconfigured_rules = [];
+        var duplicate_rule_group = [];
+        $scope.duplicate_rule_keys = [];
```

Comment:
maybe make use of the misconfigured rules somehow? so you can easily make use of the `show_configuration_errors` method.

Reply:
As discussed, the `show_configuration_errors` method is broken. Not incorporating into that right now makes the most sense. I'll be cleaning that up soon